test/ASTGen/verify-parse.swift

```diff
@@ -1,33 +1,74 @@
-// RUN: %target-run-simple-swift(-enable-experimental-feature SwiftParser -enable-experimental-feature ParserASTGen)
-// RUN: %target-run-simple-swift(-enable-experimental-feature ASTGenTypes)
+
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend %s -dump-parse -disable-availability-checking -enable-experimental-feature ParserASTGen > %t/astgen.ast
+// RUN: %target-swift-frontend %s -dump-parse -disable-availability-checking > %t/cpp-parser.ast
+// RUN: %diff -u %t/astgen.ast %t/cpp-parser.ast
```

Comment:
This is of course ersatz — we ought to implement an AST matcher.

Reply:
We should, but it's a big task and the approach you're using here can provide real value along the way.